src/components/StatusDashboard/current_migrations.jsx

```diff
@@ -59,9 +61,10 @@ export default function CurrentMigrations({ onLoad }) {
       return { ...prev, collapsed: updated };
     });
   useEffect(fetchContent(onLoad, setState), []);
-  const { closed, longterm, regular } = state;
-  const total = closed.length + longterm.length + regular.length;
-  return (
+  const { closed, longterm, paused, regular } = state;
+  const total = closed.length + longterm.length + paused.length + regular.length;
+  const fetched = total > 0;
```

Comment:
Cool trick! I had assumed that the four tables are fetched in parallel but looks like the way `fetchContent` is written, it's blocking til all four JSONs have been obtained? Not a problem for this PR, though.

Reply:
Because those requests are to static assets and should come back quickly, the fetching is hidden behind one single promise. They are *fetched* in parallel but rendering awaits all of the promises resolving before it proceeds.